users/vieting/experiments/switchboard/hybrid/feat/baseline_args.py

```diff
@@ -23,21 +23,26 @@
     get_funcs_jingjing,
 )
 
+from .specaug_sorted import ( 
+    specaug_layer_sorted,
+    get_funcs_specaug,
+)
+
 RECUSRION_LIMIT = """
 import sys
 sys.setrecursionlimit(3000)
 """
 
 
-def get_nn_args(nn_base_args, num_epochs, evaluation_epochs=None, prefix=""):
+def get_nn_args(nn_base_args, num_epochs, evaluation_epochs=None, prefix="", specaug_mask_sorting=False):
```

Comment:
It would be sufficient to add `specaug_mask_sorting` to `returnn_args` and only introduce this as input arg in `get_returnn_config()`.